Comments/src/items/VCommentBrowser.h

```diff
@@ -54,6 +54,9 @@ class COMMENTS_API VCommentBrowser : public Super<Visualization::Item>
 
 		QGraphicsWebView* browser() const;
 
+	public slots:
```

Comment:
Please do not use the `slots` keyword. Since Qt 5 there are alternative ways to connect to a function. In particular, in the place where you call connect, you can use a lambda that captures 'this' and calls the `requestUpdate` method. See here for more info: https://wiki.qt.io/New_Signal_Slot_Syntax.

Also you don't actually need a request update function with this new approach. Simple call `setUpdateNeeded` directly in the lambda.

This also removes the need to inherit from QObject. Multiple inheritance is generally a bad idea.

So in the end, the header file will not change at all.